prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -332,42 +332,46 @@ class FHIRRouter(
             // JURIS FILTER
             //  default: allowNone
             var passes = evaluateFilterCondition(getJurisFilters(receiver, orgFilters), bundle, false)
+            if (!passes) {
+                return@forEach
```

Comment:
Multiple returns can result in spaghetti code, especially if methods get longer and more complicated. Please do not do internal returns unless absolutely necessary for a programmatic reason. Add in additional 'if' statements where needed to not run logic unless needed.

This previously had short circuit logic but that was removed during a previous merge.